src/Mono.Android/Mono.Android.targets

```diff
@@ -388,4 +381,19 @@
     />
   </Target>
 
+  <Target Name="_FindDocSourceFiles">
+    <ItemGroup>
+      <_MsxDocSourceFile Include="$(XamarinAndroidSourcePath)external\android-api-docs\**" />
```

Comment:
This should probably be `…\android-api-docs\docs\Mono.Android\en`, because:

 1. `_ExportMsxDoc` won't use any of the other files.
 2. It'll be *slightly* fewer files to glob (8760 vs. 9914)

I *do* wonder how long it'll take to fill the item group, but I don't know of a better solution…

Reply:
Good catch, I updated this to use the more specific path.